src/map/map.c

```diff
@@ -5425,6 +5425,30 @@ static bool map_zone_mf_cache(int m, char *flag, char *params)
 			else if( map->list[m].flag.nocashshop )
 				map_zone_mf_cache_add(m,"nocashshop");
 		}
+	} else if(!strcmpi(flag, "nostorage")) {
+		if( !state ) {
+			if( map->list[m].flag.nostorage != 0 ) {
+				sprintf(rflag,"nostorage\t%d",map->list[m].flag.nostorage);
```

Comment:
wrong format string. nostorage flag is unsigned, then need use ``%u`` and not ``%d``